docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -663,10 +683,10 @@ You can disable token encryption in the session cookie by setting `quarkus.oidc.
 [[custom-token-state-manager]]
 ==== Session cookie and custom TokenStateManager
 
-Register a custom `io.quarkus.oidc.TokenStateManager' implementation as an `@ApplicationScoped` CDI bean if you need to customize the way the tokens are associated with the session cookie.
+If you want to customize the way the tokens are associated with the session cookie, register a custom `io.quarkus.oidc.TokenStateManager' implementation as an `@ApplicationScoped` CDI bean.
 
 For example, you might want to keep the tokens in a cache cluster and have only a key stored in a session cookie.
-Note that this approach might introduce some challenges if you need to make the tokens available across multiple microservices nodes.
+If you need to make the tokens available across multiple microservices nodes, not that this approach might introduce some challenges.
```

Comment:
The removed line reads quite well IMHO, is it not quite correct ? Minor typo in `not` but I'd rather keep the original sentence if possible